apps/omg_watcher/test/fixtures.exs

```diff
@@ -90,7 +90,7 @@ defmodule OMG.Watcher.Fixtures do
       config :omg_db, path: "#{db_path}"
       # this causes the inner test child chain server process to log info. To see these logs adjust test's log level
       config :logger, level: :info
-      config :omg_child_chain, fee_adapter_opts: [specs_file_name: "#{fee_file}"]
+      config :omg_child_chain, fee_adapter_opts: [specs_file_path: "#{fee_file}"]
```

Comment:
in the last week, this configuration has been changed 4 times :D

Reply:
Yeah the other big change to fees was from https://github.com/omisego/elixir-omg/pull/1373 which turned the file-only into an adapter for file & feed. I had to rebase and fix conflicts a few times 🤦‍♂